### PR TITLE
Article : Data Structures : Linked Lists

### PR DESCRIPTION
**Pull request by jainaman224**
#1044

### CODE REVIEW

**Comment by koustuvsinha** [comment]
Awesome article. @alayek @atjonathan please have a look, i think from the last PR lots of suggestions are already implemented.

# LGTM :+1:

File: Data-Structure-Linked-Lists.md [hunk 1]
```diff
@@ -0,0 +1,468 @@
...
+            self.head = current.get_next()
+            print("Node with data " + str(data) + " is deleted successfully")
+        else:
+            previous.set_next(current.get_next())
+            print("Node with data " + str(data) + " is deleted successfully")
+
+```
+
+:rocket: [Run Code](https://repl.it/CVq3/2)
+
```
Comment (atjonathan) [inline Data-Structure-Linked-Lists.md]: Remove this line

File: Data-Structure-Linked-Lists.md [hunk 2]
```diff
@@ -0,0 +1,468 @@
...
+    // Free space used by deleted node
+    current = NULL;
+    delete current;
+    std::cout << "Deleted succesfully" << std::endl;
+}
+
+```
+
+:rocket: [Run Code](https://repl.it/CXVt/1)
+
```
Comment (atjonathan) [inline Data-Structure-Linked-Lists.md]: Remove this line

**Comment by atjonathan** [comment]
### **LGTM :+1:**
Wanna see feedback from @alayek :facepunch:

File: Data-Structure-Linked-Lists.md [hunk 3]
```diff
@@ -0,0 +1,468 @@
+# Linked list
```
Comment (Rafase282) [inline Data-Structure-Linked-Lists.md]:
```diff
-# Linked list
+# Data Structure Linked List
```

**Comment by Rafase282** [comment]
Fixed on #1077

**Pull request merged**